spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -253,6 +270,29 @@ def main(argv: Sequence[str]):
         # Use the result of the current model as additional input of the next model
         fname_prior = fname_seg
 
+    if arguments.qc is not None:
+        # Models can have multiple input images -- create 1 QC report per input image.
+        # Models can also either have 1 OR 2 outputs per input, so we may need to split output_filenames into 2 lists
+        if len(output_filenames) == len(input_filenames):
+            iterator = zip(input_filenames, output_filenames, [None] * len(input_filenames))
+        else:
+            assert len(output_filenames) == 2 * len(input_filenames)
+            iterator = zip(input_filenames, output_filenames[0::2], output_filenames[1::2])
+
+        # Create one QC report per input image, with one or two segs per image
+        species = 'mouse' if any(s in arguments.task[0] for s in ['mouse', 'mice']) else 'human'  # used for resampling
```

Comment:
This made me realize that we use both `mouse` and `mice` across model names. :sweat_smile: